cuda_bindings/cuda/bindings/_internal/cufile_linux.pyx

```diff
@@ -43,10 +43,10 @@ cdef int get_cuda_version():
         raise NotSupportedError(f'CUDA driver is not found ({err_msg.decode()})')
     cuDriverGetVersion = dlsym(handle, "cuDriverGetVersion")
     if cuDriverGetVersion == NULL:
-        raise RuntimeError('something went wrong')
+        raise RuntimeError('Did not find cuDriverGetVersion symbol in libcuda.so.1')
```

Comment:
Tiny change to make sure this will remain accurate in the future:

Above:

```
    libcuda_so = 'libcuda.so.1'
    handle = dlopen(libcuda_so, ...)
```

Then here: `f'... {libcuda_so}'`

Reply:
This is moot if we nuke the entire function. See the linked issue and my comment below.